code/modules/organs/external/standard.dm

```diff
@@ -38,6 +38,10 @@
 		return
 	. = ..()
 
+//Can't drop root limb
+/obj/item/organ/external/chest/is_droppable()
```

Comment:
Will this interfere with the amputation mechanic that destroys the human mob when all limbs other than the chest are removed?

Reply:
That has nothing to do with that. That mechanic is handled in its own conditional that checks if only a single limb is left then gib:

_external.dm, line 1001
`	owner.remove_organ(src, TRUE, FALSE, ignore_children)
	if(istype(last_owner) && !QDELETED(last_owner) && LAZYLEN(last_owner.get_external_organs()) <= 1)
		last_owner.physically_destroyed(FALSE, disintegrate)`

This is solely to categorize things as droppable or not, so that when we gib the mob, or individual limbs, its not going to drop say the chest or stumps. I made a proc for it so its no hardcoded and doing type checks or expecting things to get deleted in time at the same time the mob is gibbed like it used to, since it was unreliable after testing.